.travis.yml

```diff
@@ -1,6 +1,16 @@
 language: node_js
 node_js:
-  - "0.10"
-  - "0.11"
-  - "0.12"
-
+  - "10.15.3"
+dist: trusty
+cache: npm
+before_install:
```

Comment:
is this necessary to do at all? npm is already installed by default in this environment

Reply:
that's assuming you don't need a specific npm version (i never have needed to do this)

Reply:
We are now defaulting to env's npm version. `before_install` block kept just spit out npm version info